src/bus/index.browser.js

```diff
@@ -0,0 +1,46 @@
+import Channels from './channels';
+
+/**
+ * Module that provides access to the message bus. This is for browser
+ * environments. Documentation for Node environments is found under `Bus`.
+ *
+ * @alias BrowserBus
```

Comment:
Why alias as BrowserBus instead of just naming BrowserBus? Is the intention to allow current imports to switch over to the .browser version without any further modifications?

Reply:
The intent is 100% to have the current imports switch over -- an end user's Webpack/Browserify config should be picking it up. More info on this: https://github.com/defunctzombie/package-browser-field-spec